### ISSUE TITLE
Django-RQ already uses sync transport for Sentry

### ISSUE THREAD

**Issue by justinmayer** [ui/django-rq#271]
The [Configuring Logging](https://github.com/ui/django-rq/blob/master/README.rst#configuring-logging) section of the README currently says:

> Please configure Raven to use sync+https:// or requests+https:// transport in settings.py [...]

But that configuration produces a deprecation warning:

> UserWarning: Transport selection via DSN is deprecated. You should explicitly pass the transport class to Client() instead.

As far as explicitly passing the `sync` transport class to Client(), it seems that [Django-RQ already does that](https://github.com/ui/django-rq/blob/3555cb5f964782c672d36f36d744e89e73d48620/django_rq/management/commands/rqworker.py#L103). (`HTTPTransport` is the [sync transport](https://docs.sentry.io/clients/python/transports/#sync).)

In short, perhaps I'm missing something, but it seems to me that this bit of documentation is out-of-date and should be updated or removed as appropriate.

**Comment by justinmayer** [comment]
@efi-mk / @selwin: What do you think?

**Comment by selwin** [comment]
@justinmayer PR welcome :)

**Comment by djmaze** [comment]
Hi. While we are at it, the information about the usage of Raven (for Sentry) is outdated as well.

**Comment by selwin** [comment]
Yeah, RQ already switched to using `sentry-sdk` in version 1.0. Can someone please open a PR to fix this?

**Comment by kichawa** [comment]
PR was merged: https://github.com/rq/django-rq/pull/425.
We look forward to the release!
